src/styles/mixins/_grid.scss

```diff
@@ -1,4 +1,5 @@
-main {
+main,
+form {
```

Comment:
Waarom is dit eigenlijk nodig, het `form` staat toch altijd in `main`? Of is dat niet zo...?

Reply:
Dit was een fix voor iets wat later aangepast is. Ik haal het weg.

Reply:
ok